src/components/grader/submissions/ImportSubmissions.js

```diff
@@ -88,43 +87,43 @@ const ImportSubmissions = (props) => {
                 </Header >
 
                     <Grid >
-                        <GridColumn style={{marginLeft:"27%", marginRight:"20%"}}>
-                        <div class="ui buttons">
-                            <GoogleLogin
-                                clientId="653543257974-p3uuv08hcftdhkolqpl2hpbbta2d1ck2.apps.googleusercontent.com"
-                                scope="https://www.googleapis.com/auth/spreadsheets.readonly https://www.googleapis.com/auth/drive.readonly"
-                                render={renderProps => (
-                                    <Button onClick={renderProps.onClick}
-                                        disabled={renderProps.disabled}
-                                        style={{ backgroundColor: '#2f2f2f', color: 'white' }}>
-
-                                        <Icon name='google' />
+                        <GridColumn style={{ marginLeft: "27%", marginRight: "20%" }}>
+                            <div class="ui buttons">
+                                <GoogleLogin
+                                    clientId="653543257974-p3uuv08hcftdhkolqpl2hpbbta2d1ck2.apps.googleusercontent.com"
+                                    scope="https://www.googleapis.com/auth/spreadsheets.readonly https://www.googleapis.com/auth/drive.readonly"
+                                    render={renderProps => (
```

Comment:
is this a specific key for the application? is it okay if anyone can use this id?

if it's okay then leave it as is, otherwise it should be moved to an environment variable